sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -962,7 +915,7 @@ def test_dict_learning_online_numerical_consistency(method):
     # as long as holding UV.
     # So here UV, ||U||_1,1 and sum(||V_k||_2) are verified
     # instead of comparing directly U and V.
-    assert_allclose(np.matmul(U_64, V_64), np.matmul(U_32, V_32), rtol=rtol)
+    assert_allclose(np.matmul(U_64, V_64), np.matmul(U_32, V_32), rtol=1e-3)
```

Comment:
Not sure why this test case failed with `rtol=1e-4`. There is 1 out of 80 elements in the array that has relative error exceeding `1e-4`.